includes/class-sensei-learners-admin-bulk-actions-controller.php

```diff
@@ -264,7 +264,7 @@ public function get_view() {
 	public function learners_admin_menu() {
 		global $menu;
 		if ( current_user_can( 'manage_sensei_grades' ) ) {
-			$learners_page = add_submenu_page( 'sensei', 'Learner Admin', 'Learner Admin', 'manage_sensei_grades', 'sensei_learner_admin', array( $this, 'learner_admin_page' ) );
+			$learners_page = add_submenu_page( 'sensei', __( 'Learner Admin', 'sensei-lms' ), __( 'Learner Admin', 'sensei-lms' ), 'manage_sensei_grades', 'sensei_learner_admin', array( $this, 'learner_admin_page' ) );
```

Comment:
It seems that this is no longer used internally (see the `hook` function above to see where it was used previously). I wonder if it would be better to deprecate this function instead?

Reply:
I noticed that, too. I think let's do it in a different PR, though.